pkg/controller/core.oam.dev/v1alpha2/application/application_controller_test.go

```diff
@@ -185,7 +183,7 @@ var _ = Describe("Test Application Controller", func() {
 				},
 			},
 			"spec": map[string]interface{}{
-				"replicaCount": int64(2),
+				"replicaCount": float64(2),
```

Comment:
why the type changed?

Reply:
Before, this assertion uses gomega.BeEquivalentTo to compare two objects, but this assertion will ingore some types check, such as int v.s. float. Now, I use cmp.Diff to compare to get more readable diff info. cmp.Diff will not ignore type differences.
Actually, json.Unmarshal will always treat numbers as float64 but not int64.